apps/dashboard/pages/donations.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC, useEffect, useState } from 'react'
+
+import { useUpdateEffect } from '@chakra-ui/react'
+import { GetStaticPropsContext, InferGetStaticPropsType } from 'next'
+import { useRouter } from 'next/router'
+
+import { useStrapiRequest } from '@wsvvrijheid/services'
+import { ssrTranslations } from '@wsvvrijheid/services/ssrTranslations'
+import { Donation, StrapiLocale } from '@wsvvrijheid/types'
+import {
+  AdminLayout,
+  DataTable,
+  PageHeader,
+  donationColumns,
+} from '@wsvvrijheid/ui'
+
+type PageProps = InferGetStaticPropsType<typeof getStaticProps>
+
+const DonationsPage: FC<PageProps> = ({ seo }) => {
+  const [currentPage, setCurrentPage] = useState<number>(1)
+  const [searchTerm, setSearchTerm] = useState<string>()
+
+  const [sort, setSort] = useState(['createdAt:desc'] as any)
```

Comment:
You can merge after this adjustment

```suggestion
  const [sort, setSort] = useState<Sort>(['createdAt:desc'])
```